R/plumber.R

```diff
@@ -521,12 +521,9 @@ plumber <- R6Class(
 
     route = function(req, res) {
       getHandle <- function(filt) {
-        handlers <- private$ends[[filt]]
-        if (!is.null(handlers)) {
-          for (h in handlers) {
-            if (h$canServe(req)) {
-              return(h)
-            }
+        for (h in private$ends[[filt]]) {
+          if (h$canServe(req)) {
+            return(h)
```

Comment:
I'm going to revert this, then LGTM to merge

Reply:
It's been a long time and I'm not 100% sure why I made the change at the time, so feel free.